### DISCUSSION TITLE
2023년 03월 15일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#7] [General]
# 2023년 03월 15일

[ 2023-03-01 ] [ 2023-03-02 ] [ 2023-03-05 ] [ 2023-03-06 ] [ 2023-03-07 ] [ 2023-03-08 ] [ 2023-03-09 ] [ 2023-03-10 ] [ 2023-03-12 ] [ 20…

https://cyma-s.github.io/2023-03-14/

**Comment by poi1649** [comment]
어제의 나도 구체적이지 않으면 어떡하죠?

**Reply by Cyma-s** [reply]
그럴 땐 포이는 힙합을 춰

**Comment by drunkenhw** [comment]
주말에 생일이신가봐요~ 생일 축하드려요~

**Reply by Cyma-s** [reply]
감사합니다 ^ㅅ^ ~~
